modules/claims_api/app/models/claims_api/veteran.rb

```diff
@@ -68,6 +68,10 @@ def mpi
       @mpi ||= MPIData.for_user(self)
     end
 
+    def mpi_no_cache
+      MPIData.for_user(self)
+    end
```

Comment:
I did test trying to call the above method after `add_person_proxy` and it did not contain the new value so this does appear to be a need, whether handled like this or some other way.